infocenter-web/infocenter-product/infocenter.product

```diff
@@ -37,12 +37,12 @@ org.osgi.framework.bootdelegation=*
       <plugin id="com.ibm.icu"/>
       <plugin id="jakarta.servlet-api"/>
       <plugin id="javax.el"/>
-      <plugin id="javax.servlet.jsp-api"/>
+      <plugin id="javax.servlet.jsp"/>
       <plugin id="javax.xml"/>
       <plugin id="org.apache.felix.scr"/>
       <plugin id="org.apache.jasper.glassfish"/>
-      <plugin id="org.apache.lucene.analysis-common"/>
-      <plugin id="org.apache.lucene.analysis-smartcn"/>
+      <plugin id="org.apache.lucene.analyzers-common"/>
```

Comment:
These are the old lucene ids thus not ok.

Reply:
That was not intended and was the result of an erroneous rebase.